TypeScript error in @sentry/integrations introduced in 7.16.0

### Is there an existing issue for this?

- [X] I have checked for existing issues https://github.com/getsentry/sentry-javascript/issues
- [X] I have reviewed the documentation https://docs.sentry.io/
- [X] I am using the latest SDK release https://github.com/getsentry/sentry-javascript/releases

### How do you use Sentry?

Sentry Saas (sentry.io)

### Which package are you using?

@sentry/serverless

### SDK Version

7.18.0

### Framework Version

Node 16, TypeScript 4.7

### Link to Sentry event

_No response_

### Steps to Reproduce

- upgrade Sentry packages to 7.16.0+
- make use of `import { ExtraErrorData } from "@sentry/integrations";`
- compile with `tsc`

Note: this appears to be related to #6016 but affects `@sentry/integrations` rather than `@sentry/utils`.

### Expected Result

No error

### Actual Result

```
node_modules/@sentry/integrations/types/offline.d.ts:2:71 - error TS2304: Cannot find name 'Window'.

2 export declare const WINDOW: import("@sentry/utils").InternalGlobal & Window;
                                                                        ~~~~~~

node_modules/@sentry/integrations/types/reportingobserver.d.ts:2:71 - error TS2304: Cannot find name 'Window'.

2 export declare const WINDOW: import("@sentry/utils").InternalGlobal & Window;
                                                                        ~~~~~~
```

--- a/packages/integrations/src/offline.ts
+++ b/packages/integrations/src/offline.ts
@@ -4,7 +4,7 @@ import { Event, EventProcessor, Hub, Integration } from '@sentry/types';
 import { GLOBAL_OBJ, logger, normalize, uuid4 } from '@sentry/utils';
 import localForage from 'localforage';
 
-export const WINDOW = GLOBAL_OBJ as typeof GLOBAL_OBJ & Window;
+const WINDOW = GLOBAL_OBJ as typeof GLOBAL_OBJ & Window;
 
 type LocalForage = {
   setItem<T>(key: string, value: T, callback?: (err: any, value: T) => void): Promise<T>;

--- a/packages/integrations/src/reportingobserver.ts
+++ b/packages/integrations/src/reportingobserver.ts
@@ -1,7 +1,7 @@
 import { EventProcessor, Hub, Integration } from '@sentry/types';
 import { GLOBAL_OBJ, supportsReportingObserver } from '@sentry/utils';
 
-export const WINDOW = GLOBAL_OBJ as typeof GLOBAL_OBJ & Window;
+const WINDOW = GLOBAL_OBJ as typeof GLOBAL_OBJ & Window;
 
 interface Report {
   [key: string]: unknown;

